cms/djangoapps/import_from_modulestore/models.py

```diff
@@ -0,0 +1,172 @@
+"""
+Models for the course to library import app.
+"""
+
+import uuid as uuid_tools
+from typing import Self, Optional
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import (
+    LearningContextKeyField,
+    UsageKeyField,
+)
+from openedx_learning.api.authoring_models import LearningPackage, PublishableEntity
+
+from .data import ImportStatus
+
+User = get_user_model()
+
+
+class Import(TimeStampedModel):
+    """
+    Represents the action of a user importing a modulestore-based course or legacy
+    library into a learning-core based learning package (today, that is always a content library).
+    """
+
+    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False, unique=True, db_index=True)
```

Comment:
```suggestion
    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False, unique=True)
```
adding a unique constraint implicitly creates a DB index, so you can omit db_index.

Reply:
Thanks, removed.